.changeset/flat-snakes-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: [PLACEHOLDER]
```

Comment:
@Mouvedia Feel free to suggest your idea here. I'll commit. 👍🏼

Reply:
I wanted to add:

> [fork({ cssWideKeywords: ['-moz-initial'] })](https://github.com/csstree/csstree/pull/243#issuecomment-2452852431)

Reply:
OK. Feel free to open a follow-up PR because I'll merge this right now.

I think it's easier for us to trace commits/PRs by separating just updating a dependency from fixing a bug.